download_tokenizer_files.py

```diff
@@ -16,6 +16,7 @@
     "Xenova/gpt-4o",
     "Xenova/claude-tokenizer",
     "Qwen/Qwen2.5-Coder-32B-Instruct",
+    "Xenova/gemini-nano",
```

Comment:
Hi @zakiali would you please review this to check whether this file is correct? Thank you

Reply:
Doing a bit googling, it looks like gemini 1.5 flash uses the same tokenizer as gemma 2, so we can probably use this instead https://huggingface.co/Xenova/gemma-2-tokenizer